doc/users/next_whats_new/legend_title_fontproperties_kwarg.rst

```diff
@@ -0,0 +1,15 @@
+Legend now has a title_fontproperties kwarg
+-------------------------------------------
+
+The title for a `.Figure.legend` and `.Axes.legend` can now have its
+title's font properties set via the ``title_fontproperties`` kwarg, defaults 
+to ``None``, which means the legend's title will have the font properties
+as set by the :rc:`legend.title_fontsize`. Through this new kwarg, one
+can set the legend's title font properties through either FontProperties
+or dict, for example:
```

Comment:
```suggestion
title's font properties set via the ``title_fontproperties`` keyword
argument, for example:
```
This description should only give the user an idea about the new feature. Details like supported formats and defaults are not relevant here. They can be looked up easily using the links above.